src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
+
+use crate::prelude::*;
```

Comment:
The scene is not tested and its not working correctly. I'm assuming its not tested because even panning and zooming break the layout, which is a very basic action in graph editor:
    ![2020-08-06 10 17 08](https://user-images.githubusercontent.com/1623053/89508976-f876ee00-d7ce-11ea-85fa-e88d23232769.gif)